controllers/account/controllers/account_controller.go

```diff
@@ -151,6 +160,36 @@ func (r *AccountReconciler) syncAccount(ctx context.Context, name, accountNamesp
 	if err := r.syncRoleAndRoleBinding(ctx, name, userNamespace); err != nil {
 		return nil, fmt.Errorf("sync role and rolebinding failed: %v", err)
 	}
+
+	// add account balance when account is new user
+	if key, ok := account.Annotations[AccountAnnotationNewAccount]; !ok || key != "false" {
+		stringAmount := os.Getenv(NEWACCOUNTAMOUNTENV)
+		if stringAmount != "" {
```

Comment:
Can changed to "if == "" && continue" code style ,to have better cleaner code review.